Makefile.config.in

```diff
@@ -182,6 +182,9 @@ INSTALL_SOURCE_ARTIFACTS=@install_source_artifacts@
 OC_CFLAGS=@oc_cflags@
 CFLAGS?=@CFLAGS@
 OC_CPPFLAGS=-I$(ROOTDIR)/runtime @oc_cppflags@
+OC_NATIVE_CPPFLAGS=\
+  -DNATIVE_CODE -DTARGET_$(ARCH) -DMODEL_$(MODEL) -DSYS_$(SYSTEM)
+OC_ASPPFLAGS=-I $(ROOTDIR)/runtime $(OC_NATIVE_CPPFLAGS)
```

Comment:
Both of these in `Makefile.common`, rather than `Makefile.config.in`? They relate to how the compiler (well, runtime) is built internally, and don't need to be "installed"?